tools/payments/client.go

```diff
@@ -149,7 +149,7 @@ func (rc *redisClient) PrepareTransactions(ctx context.Context, signer httpsigna
 		stream := payments.PreparePrefix + payoutID
 		err := rc.redis.AddMessages(ctx, stream, messages...)
 		if err != nil {
-			return fmt.Errorf("failed to exec prepare transaction commands: %w", err)
+			return fmt.Errorf("failed to enqueue transactions to redis: %w", err)
```

Comment:
Non-blocking nit: Here and below, the `to redis` thing seems like an unnecessary detail.